stumptown.py

```diff
@@ -33,7 +35,11 @@ def scrape():
                 status = 'Sold Out'
             else:
                 status = 'Available'
-            coffee = Coffee(name=name, roaster=roaster, description=description, price=price, notes =notes, region=region, status=status)
+            # size in ounces
+            size = coffee_soup.find('div', {'class':'product _specs'}).find_all('p')[1].string.split()[2]
+            coffee = Coffee(name=name, roaster=roaster, description=description, 
+                price=price, notes=notes, region=region, status=status,
+                product_page=product_url, size=size)
```

Comment:
Maybe do a check on the coffee name and roaster before you put? We'll prob get a lot of dupes if we do it that way. If you were planning to do this in another PR, ignore me :)